circuits/comparators.circom

```diff
@@ -98,6 +98,23 @@ template LessThan(n) {
     out <== 1-n2b.out[n];
 }
 
+// Do Range checks on the inputs to avoid overflow
+template SafeLessThan(n) {
+    assert(n <= 252);
+    signal input in[2];
+    signal output out;
+
+    component aInRange = Num2Bits(252);
+    aInRange.in <== in[0];
+    component bInRange = Num2Bits(252);
+    bInRange.in <== in[1];
+
+    component n2b = Num2Bits(n+1);
+
+    n2b.in <== in[0]+ (1<<n) - in[1];
+
+    out <== 1-n2b.out[n];
```

Comment:
Perhaps we could use a normal `LessThan` component here to avoid code reuse. It makes it clearer what the difference between `LessThan` and `SafeLessThan` is, and if `LessThan` changes in the future we don't risk two difference implementations.

Reply:
Agree, applied change in this [commit](https://github.com/iden3/circomlib/pull/86/commits/4309a3c10c2ebb73e81735ffcf12d1711f11a431)